draft-ietf-httpbis-priority.md

```diff
@@ -562,8 +562,9 @@ respond with a connection error of type PROTOCOL_ERROR.
 If a PRIORITY_UPDATE frame is received with a Prioritized Stream ID of 0x0, the
 recipient MUST respond with a connection error of type PROTOCOL_ERROR.
 
-If a client receives a PRIORITY_UPDATE frame, it MUST respond with a connection
-error of type PROTOCOL_ERROR.
+PRIORITY_UPDATE frames are only sent by clients. If a client receives a
```

Comment:
```suggestion
Servers MUST NOT send PRIORITY_UPDATE frames. If a client receives a
```

How about being more direct? It has been our position that it is a fact that servers do not send PRIORITY_UPDATE frames, because we say that they are sent by "client only." But what led to this PR is the argument that that might be clear enough. Then, I think we'd better use a MUST NOT to be absolutely clear, rather than implying by using the word "only."

Reply:
This would accurately reflect the intent that we've had for the frame. Seems like it's a borderline normative change, so we might want to also let the WG know before landing this.